src/Stratis.Bitcoin.Tests/Utilities/DateTimeProviderTest.cs

```diff
@@ -12,7 +12,7 @@ public void GetUtcNowReturnsCurrentUtcDateTime()
         {
             DateTime result = DateTimeProvider.Default.GetUtcNow();
 
-            Assert.Equal(DateTime.UtcNow.ToString("yyyyMMddhhmmss"), result.ToString("yyyyMMddhhmmss"));
+            Assert.Equal(DateTimeProvider.Default.GetUtcNow().ToString("yyyyMMddhhmmss"), result.ToString("yyyyMMddhhmmss"));
```

Comment:
I'm not sure these changes are correct, you're testing the same values here.

Reply:
Good spot, fixed